### PR TITLE
Use post-process material domain for FXAA

### PR DESCRIPTION
**Pull request by bejado**
Convert the FXAA to a post-process material. This also removes:
- `setSource` method, which isn't needed anymore
- The `uvScale` uniform, which was always 1.0

### CODE REVIEW

File: filament/CMakeLists.txt [hunk 1]
```diff
@@ -135,6 +135,9 @@ set(PRIVATE_HDRS
         src/UniformBuffer.h
         src/upcast.h)
 
+# tonemapping.mat and fxaa.mat #include files inside the shaders/src directory. Since the
```
Comment (prideout) [inline filament/CMakeLists.txt]: Nice comment

**Comment by bejado** [comment]
@prideout Nope, working on a PR to deprecate all the "old" post-process stuff.

File: filament/src/PostProcessManager.cpp [hunk 2]
```diff
@@ -235,22 +204,29 @@ FrameGraphId<FrameGraphTexture> PostProcessManager::fxaa(FrameGraph& fg,
                 });
                 data.rt = builder.createRenderTarget(data.output);
             },
-            [=](FrameGraphPassResources const& resources,
+            [=, engine = &engine](FrameGraphPassResources const& resources,
```
Comment (pixelflinger) [inline filament/src/PostProcessManager.cpp]: couldn't you just write `[=, &engine]` here?

Comment (bejado) [reply filament/src/PostProcessManager.cpp]: Yes, good catch

**Pull request merged**